### PR TITLE
improved readme

### PR DESCRIPTION
**Pull request by SophieDeBenedetto**
@jmburges

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,14 @@
+# Object Oriented Email Parser
 
+## Objectives
 
-# Object Oriented Email Parser
+1. Practice defining a class and using an attr_accessor to create setter and getter methods
```
Comment (jmburges) [inline README.md:5]: backticks

File: README.md [hunk 2]
```diff
@@ -18,12 +22,16 @@ parser.parse
...
 
 ## Instructions
 
-Get all the tests to pass by implementing an "EmailParser" class.
+This lab is test-driven, so run the test suite to get started and used the test output to get the program working. 
+
+**Hints:**
+
+* How will you control for parsing a list of emails that is *either* comma separated *or* separated by a white space. 
+* Use an attr_accessor to set and get the list of emails
```
Comment (jmburges) [inline README.md:36]: backticks around `attr_accessor`

**Comment by jmburges** [comment]
minor typos and :shipit:

File: README.md (hunk 2 above)
Comment (SophieDeBenedetto) [inline README.md:36]: fixed.

**Pull request merged**